Makefile

```
@@ -15,14 +15,17 @@ help:  # Display help
 all: fmt lint unit  # Run all quick, local commands
 
 fmt:  # Format the Python code
-	uv run --all-extras ruff check --fix $(PROJECT)
+	uv tool run ruff check --fix $(PROJECT)
 	uv tool run ruff format $(PROJECT)
 
-lint:  # Check for linting issues
+_ensurevenv:  # setup venv with system packages
+	[ -d $(PROJECT).venv ] || uv venv $(PROJECT).venv --no-managed-python --system-site-packages --project $(PROJECT)src
```

Should this become something like this?
```suggestion
$(PROJECT).venv: pyproject.toml
	uv venv $(PROJECT).venv --no-managed-python --system-site-packages --project $(PROJECT)src
```

---

That would say that `$(PROJECT).venv` is created by the `uv venv…` command, and depends on `pyproject.toml`.

So `uv venv…` would only be invoked if `.venv` doesn't exist, or if `pyproject.toml` changes. We might need a `rm -rf $(PROJECT).venv` first, in the latter case?

---

Though with `--always-make` that probably doesn't make sense anyway.